content/docs/user-patterns/_index.md

```diff
@@ -10,7 +10,7 @@ weight: 999
 toc: false
 ---
 
-RKVST is a very flexible system, and enables users to record When Who Did almost anything to anything. To get the best out of the system, however, it is important to model your real-world assets and business processes efficiently into RKVST Assets and Events.
+RKVST is a very flexible system, and enables users to record Who Did What When almost anything to anything. To get the best out of the system, however, it is important to model your real-world assets and business processes efficiently into RKVST Assets and Events.
```

Comment:
Who did what when _to_ almost anything

Reply:
I've already merged so this will be answered in a different PR